CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Release notes for the puppetlabs-puppetdb module.
 * Drop support for Puppet 6
 * Drop support for Ubuntu 16.04 and Scientific Linux 7
 * Remove all code related to embedded database
+* Require and install postgres 14 by default
```

Comment:
we don't need to update the changelog here, the release prep task will rewrite it.

Reply:
will drop the commit after all tests pass